docs/CONFIGURATION.md

```diff
@@ -440,6 +440,16 @@ Once the callback complete, the worker will be signaled with `SIGKILL`.
 This callback being called in an indication that something is preventing the
 soft timeout from working.
 
+### `before_worker_exit`
+
+Called in the monitor process before a worker is shut down.
```

Comment:
I don't think this is correct. It's called in the worker process.

Reply:
You are correct. I forgot to notice where the `fork` happens 🤦🏼 

I’ve pushed a fix.